easybuild/easyconfigs/b/BBMap/BBMap-35.82-foss-2015b-Java-1.8.0_66.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '35.82'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = """BBMap short read aligner, and other bioinformatic tools."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+java = 'Java'
+javaver = '1.8.0_66'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver, '', True)]
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz' ]
```

Comment:
@nathanhaigh: last remark: please move this up, below `toolchain`, to be consistent with other easyconfigs